src/content/components/FlameChartCanvas.js

```diff
@@ -34,68 +40,18 @@ const ROW_HEIGHT = 16;
 const TEXT_OFFSET_START = 3;
 const TEXT_OFFSET_TOP = 11;
 
-class FlameChartCanvas extends Component {
+class FlameChartCanvas extends PureComponent {
 
-  _requestedAnimationFrame: boolean
-  _devicePixelRatio: number
-  _textMeasurement: null|TextMeasurement
-  _ctx: null|CanvasRenderingContext2D
+  _textMeasurement: null | TextMeasurement
 
   props: Props
```

Comment:
Yeah, we should have a linting rule in place that covers this. I looked for one briefly but didn't find one, although maybe we can just switch to prettier.